diesel_compile_tests/tests/fail/select_requires_valid_grouping.stderr

```diff
@@ -0,0 +1,88 @@
+warning: unused `#[macro_use]` import
```

Comment:
As this warning is not relevant, could we remove the corresponding `#[macro_use]`?